### DISCUSSION TITLE
Perhaps a small communication feature idea?

### DISCUSSION THREAD

**Discussion by dontneedone2** [jomjol/AI-on-the-edge-device#1555] [Ideas]
There are already some interface options. 

I think it would be great if there were an additional option for the ESP32 itself to call up a custom URL after the measurement and transfer the values via GET. 

Please do not recommend that I use MQTT. That doesn't work with an external web server. And calling the JSON from external is not so easy either. But with a custom URL call you would have a lot of possibilities.

**Comment by caco3** [comment]
If you have a home atomation system working, you better do it there.

Eg. with Homeassistant, you could trigger an action in case of a certain MQTT message. And that action then can call any REST API.

Doing this on the device is IMO not the right place.

**Comment by dontneedone2** [comment]
For the people who have a Smarthome server running, this is of course the ideal way. For those who don't have a Pi or anything else running and don't have much of a clue, it is very inconvenient to set up something extra in their own network. 

But that was just an idea. I have built a solution for myself. 

I would have liked to install several ESPs in my circle of friends and family, which then send data centrally to my external server. I would then have taken care of the one server. But I can't install another device in every household in addition to the ESP, which I also have to take care of regularly.


Translated with www.DeepL.com/Translator (free version)

**Reply by caco3** [reply]
I see your point.

Just a note: A MQTT server could also be in the internet, there are also free servers available (just keep privacy in mind)

**Reply by dontneedone2** [reply]
hm... 🤔 good note... i have to think about it.
